### ISSUE TITLE
Switch to JavaFx

### ISSUE THREAD

**Issue by VaiTon** [UniversalMediaServer/UniversalMediaServer#2063]
Related to #741.

UMS uses java swing to provide a graphical user interface. Actually i think JavaFx would be a better toolkit. It is in fact much more capable in terms of customization and performance and it supports virtually every os natively.

**Comment by SubJunk** [comment]
Yes I'm interested in this too. I dipped my toes in when I did https://github.com/UniversalMediaServer/UniversalMediaServer/pull/1906/files and it seemed nice, but as I mentioned in the comment on there https://github.com/UniversalMediaServer/UniversalMediaServer/pull/1906#issuecomment-585087513 it wasn't ready on OpenJDK yet.

When we implement https://github.com/UniversalMediaServer/UniversalMediaServer/issues/1984 we can look at this more. I will probably do that very soon since I have been running UMS on JRE 14 locally and haven't seen any problems.
